app/main.py

```diff
@@ -1,3 +1,58 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: None) -> None:
+        if isinstance(other, Distance):
+            return Distance(round(self.km + other.km, 2))
+        elif isinstance(other, (int, float)):
+            return Distance(round(self.km + other, 2))
+
+    def __iadd__(self, other: None) -> None:
```

Comment:
The `__iadd__` method should have a return type of `Distance` instead of `None` since it returns `self`, which is a `Distance` object.